src/metabase/server/middleware/security.clj

```diff
@@ -49,8 +49,9 @@
   the original request was HTTPS; if sent in response to an HTTP request, this is simply ignored)"
   {"Strict-Transport-Security" "max-age=31536000"})
 
-(def ^:private content-security-policy-header
+(defn- content-security-policy-header
```

Comment:
why make it a function?

Reply:
Needs to be re-evaluated each time it's called so that `with-redefs` works in the tests to change between dev and prod

Reply:
Sounds good!